enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/service/CommonOnboardingService.java

```diff
@@ -97,7 +91,8 @@ public OnboardingProcessEntity updateProcess(OnboardingProcessEntity process) {
 
     @Override
     public void updateProcess(final UpdateProcessRequest request) throws OnboardingProcessException {
-        final OnboardingProcessEntity process = findProcess(request.getProcessId());
+        // The onboarding process is locked until the end of the transaction
```

Comment:
Just wondering about moving this sentence to javadoc, but not sure.

Reply:
I will use the debug logging.